forgerock-authenticator/src/main/java/org/forgerock/android/auth/Account.java

```diff
@@ -182,6 +238,9 @@ public String toJson() {
             jsonObject.put("imageURL", imageURL);
             jsonObject.put("backgroundColor", backgroundColor);
             jsonObject.put("timeAdded", timeAdded != null ? timeAdded.getTimeInMillis() : null);
+            jsonObject.put("policies", policies);
```

Comment:
I think that will treat policies as String instead of JSONObject.

Reply:
We want this treated as JSON String to make it easy to serialize to other platforms.